lib/manageiq/environment.rb

```diff
@@ -162,6 +160,18 @@ def self.system!(*args)
       options[:chdir] ||= APP_ROOT
       system(*args, options) || abort("\n== Command #{args} failed in #{options[:chdir]} ==")
     end
+
+    def self.ci?
+      ENV.fetch("CI", nil) == "true"
+    end
+
+    def self.skip_database_reset?
+      ENV.key?("SKIP_DATABASE_RESET") ? ENV.fetch("SKIP_DATABASE_RESET") : ci?
```

Comment:
I need to be able to support explicitly passing "false" here for the cypress tests, which is why I had the `if ENV.fetch("SKIP_DATABASE_RESET") == "false"` above.

Additionally, I was trying to match the pattern in manageiq/bin/setup, where we don't use ci directly, but instead set the env var and then assume it's set (this allows us to continue to use it in subsequent scripts).

So how about?

```suggestion
      ENV["SKIP_DATABASE_RESET"] ||= ci?.to_s
      ENV.fetch("SKIP_DATABASE_RESET") != "false"
```

Reply:
Another option is to just have this method only check 

```suggestion
      ENV.fetch("SKIP_DATABASE_RESET") != "false"
```

and then the defaulting is left up above on line 21 but uses the new .ci? method

Reply:
Doesn't `fetch` accept a block for the fallback value?

```suggestion
      ENV.fetch("SKIP_DATABASE_RESET") { cli? }
```

Reply:
My bad I completely forgot the `== "true"`, updated